pkgs/development/python-modules/django-markdownx/default.nix

```
@@ -0,0 +1,50 @@
+{ lib
+, buildPythonPackage
+, django
+, fetchFromGitHub
+, markdown
+, pillow
+, python
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "django-markdownx";
+  version = "4.0.7";
+  format = "setuptools";
```

```suggestion
  pyproject = true;
```
Could you please use `pyproject = true` instead?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters

---

The project has no pyproject.toml file, so we can't use that

---

see
> Note that the pyproject format falls back to using setuptools, so you can use pyproject = true even if the package only has a setup.py.